src/ui/App.swift

```diff
@@ -189,10 +189,26 @@ class App: AppCenterApplication, NSApplicationDelegate {
         KeyRepeatTimer.toggleRepeatingKeyPreviousWindow()
     }
 
-    func focusSelectedWindow(_ window: Window?) {
+    func focusSelectedWindow(_ selectedWindow: Window?) {
         hideUi()
         guard !CGWindow.isMissionControlActive() else { return }
-        window?.focus()
+        guard let window = selectedWindow else { return }
```

Comment:
Actually, this guard could be moved above `CGWindow.isMissionControlActive`. It's a bit expansive to check if Mission Control is active, so better first check if the window exists as that check is cheap.

It's not directly related to the PR, but the PR touching that area of code could be an opportunity to improve the code/performance here 👍

Reply:
I'll make this little improvement as an edit to the commit, and I'll commit it under your username